### PR TITLE
Fixed the Docker environement

### PR DESCRIPTION
**Pull request by tarekziade**
## Closes https://github.com/elastic/connectors-ruby/issues/238

We no longer have a HTTP server. Fixed the Docker environement

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,18 +35,52 @@ TBD
...
+  "name" : "my-connector",
+  "api_key" : "zIJThFg9TO6uaYVy57TFSA",
+  "encoded" : "NGVPV2dZSUJBWWRNaUdIY3htS0g6eklKVGhGZzlUTzZ1YVlWeTU3VEZTQQ=="
+}
+```
+
+And then use the `encoded` value:
+
+```
+make run-docker API_KEY=YmRhYWdZSUJLaFBUeWg3bm16Mmg6ZFh2NXBfVndSc3V3NnpBZHVRdU1xZw==
```
Comment (seanstory) [inline README.md:83]:
```suggestion
make run-docker API_KEY="NGVPV2dZSUJBWWRNaUdIY3htS0g6eklKVGhGZzlUTzZ1YVlWeTU3VEZTQQ=="
```

Comment (seanstory) [reply README.md:83]: just to make it line up with the output from the above example

File: script/run_in_docker.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+DIR="$(dirname "${BASH_SOURCE[0]}")"
+DIR="$(realpath "${DIR}")"
+
+echo $DIR
+
+
+for var in `compgen -v`
+do
+	export $var
```
Comment (seanstory) [inline script/run_in_docker.sh:11]:
```suggestion
    export $var
```
eww, tabs :P

**Review by seanstory — APPROVED** [review]
LGTM. Two small nits

**Comment by artem-shelkovnikov** [comment]
Should this PR be backported to 8.4?

**Pull request merged**